### PR TITLE
Review chapter renaming

### PR DESCRIPTION
**Pull request by aheusingfeld**
- renamed 03-introducing-the-module to 01-module-lesson1
- renamed 04-second-learning-unit to 02-module-lesson2
- added template for lessons 3-5
- Fixed broken links and formatting in lessons 1+2

### CODE REVIEW

File: docs/01-module-lesson1/00-structure.adoc
Comment (programming-wolf) [inline docs/01-module-lesson1/00-structure.adoc]: Folder name should be `01-module-lesson-one`, `02-module-lesson-two`, etc.

File: docs/01-module-lesson1/LZ-1-01.adoc
Comment (programming-wolf) [inline docs/01-module-lesson1/LZ-1-01.adoc]: see above.

File: docs/02-module-lesson2/LZ-2-01.adoc
Comment (programming-wolf) [inline docs/02-module-lesson2/LZ-2-01.adoc]: Please re-add, and change the type to `NOTE`

File: docs/03-module-lesson3/LZ-3-02.adoc
Comment (programming-wolf) [inline docs/03-module-lesson3/LZ-3-02.adoc]: R1 can (has to) be removed.

File: docs/04-module-lesson4/01-duration-terms.adoc
Comment (programming-wolf) [inline docs/04-module-lesson4/01-duration-terms.adoc]: NOTE

File: docs/04-module-lesson4/LZ-4-02.adoc
Comment (programming-wolf) [inline docs/04-module-lesson4/LZ-4-02.adoc]: R1 has to be removed (and all other `R*`)

**Review by programming-wolf — CHANGES_REQUESTED** [review]

File: docs/01-module-lesson1/00-structure.adoc
Comment (aheusingfeld) [reply docs/01-module-lesson1/00-structure.adoc]: done.

File: docs/03-module-lesson3/LZ-3-02.adoc
Comment (aheusingfeld) [reply docs/03-module-lesson3/LZ-3-02.adoc]: Done

**Comment by aheusingfeld** [comment]
@programming-wolf the CPSA-A logo needs a bit of your layout love. I just included it but it's definitely not _nice_, yet

File: docs/00-preamble/00-introduction.adoc
Comment (programming-wolf) [inline docs/00-preamble/00-introduction.adoc]: Content is not supposed to be added to the structure files (`00-some-text`). I will move it into a different file.

File: docs/01-basics/00-basics.adoc
Comment (programming-wolf) [inline docs/01-basics/00-basics.adoc]: If it is still included at this point, we should move it back to this folder. Including from a parent folder feels.. wrong.

File: docs/01-basics/00-basics.adoc
Comment (programming-wolf) [inline docs/01-basics/00-basics.adoc:21]: Why did you remove the page break?

**Review by programming-wolf — CHANGES_REQUESTED** [review]

File: docs/00-preamble/00-introduction.adoc
Comment (aheusingfeld) [reply docs/00-preamble/00-introduction.adoc]: > Content is not supposed to be added to the structure files (00-some-text).

That's definitely something you should state in the README! Headings are an exception from that rule, I assume?!

>  I will move it into a different file.

So, I assume this is not blocking the merge?! ;)

File: docs/01-basics/00-basics.adoc
Comment (aheusingfeld) [reply docs/01-basics/00-basics.adoc]: > Including from a parent folder feels.. wrong.

I had the same feeling... yet, it was not about this file but with the images for the timing pie chart. It's tricky: On the one hand, we want to make it comfortable and clear for curriculum maintainers in which places adjustments are necessary. At the same time, we don't want to make it too hard to understand where the bits and pieces should be located.

Suggestion: Let's move this file as well as the 2 images back into the "basics" folder and name it "Basics on the module {curriculum-short}"

File: docs/01-basics/00-basics.adoc
Comment (aheusingfeld) [reply docs/01-basics/00-basics.adoc:21]: ...because I couldn't see any reason to have a page break within a chapter - especially if the sub chapters are so closely related, this additional whitespace didn't feel right but rather interrupted the visual flow.

**Comment by aheusingfeld** [comment]
ping @programming-wolf

File: docs/01-basics/00-basics.adoc
Comment (programming-wolf) [reply docs/01-basics/00-basics.adoc:21]: Well, it was a visually beautiful (read: adequate) break, but ok. 🙂 Your reasoning sounds reasonable.

File: docs/01-basics/00-basics.adoc
Comment (programming-wolf) [reply docs/01-basics/00-basics.adoc]: I agree. Go for it 🙂

File: docs/00-preamble/00-introduction.adoc
Comment (programming-wolf) [reply docs/00-preamble/00-introduction.adoc]: No, it is not. I will create a new issue for that.

File: docs/01-basics/00-basics.adoc
Comment (programming-wolf) [reply docs/01-basics/00-basics.adoc]: Images have to be put into the "images" folder, because that is defined as source-folder for all images.
I created a new folder called `01-basics` in the global images folder, where these images now reside in. The adoc file was moved back to the basics folder.

**Review by programming-wolf — APPROVED** [review]

**Pull request merged**
